example/app.js

```diff
@@ -5,19 +5,21 @@ import { render } from 'react-dom';
 import { Admin, Resource } from 'admin-on-rest';
 import jsonRestClient from 'aor-json-rest-client';
 import { Delete } from 'admin-on-rest/mui';
+import { resolveBrowserLocale } from 'admin-on-rest/i18n';
```

Comment:
should be imported from `admin-on-rest`, only components get imported from a different location (because they may eventually be in another package)